crates/common/src/msgs/data_requests/sudo/mod.rs

```diff
@@ -34,40 +34,30 @@ pub struct DistributionSend {
 #[cfg_attr(feature = "cosmwasm", cosmwasm_schema::cw_serde)]
 #[cfg_attr(not(feature = "cosmwasm"), derive(serde::Serialize, Debug, PartialEq))]
 #[cfg_attr(not(feature = "cosmwasm"), serde(rename_all = "snake_case"))]
-pub enum DistributionKind {
+pub enum DistributionMessage {
     /// For burning funds
     Burn(DistributionBurn),
     /// For rewarding an executor
     ExecutorReward(DistributionExecutorReward),
-    /// For sending the funds to someone
-    Send(DistributionSend),
+    /// For rewarding a data proxy
+    DataProxyReward(DistributionDataProxyReward),
 }
 
 #[cfg_attr(feature = "cosmwasm", cosmwasm_schema::cw_serde)]
 #[cfg_attr(not(feature = "cosmwasm"), derive(serde::Serialize, Debug, PartialEq))]
 #[cfg_attr(not(feature = "cosmwasm"), serde(rename_all = "snake_case"))]
-pub enum DistributionType {
-    TallyReward,
-    ExecutorReward,
-    TimedOut,
+pub enum RefundType {
+    Timeout,
     NoConsensus,
-    RemainderRefund,
-}
-#[cfg_attr(feature = "cosmwasm", cosmwasm_schema::cw_serde)]
-#[cfg_attr(not(feature = "cosmwasm"), derive(serde::Serialize, Debug, PartialEq))]
-#[cfg_attr(not(feature = "cosmwasm"), serde(rename_all = "snake_case"))]
-pub struct DistributionMessage {
-    pub kind:  DistributionKind,
-    #[serde(rename = "type")]
-    pub type_: DistributionType,
+    Remainder,
```

Comment:
I think this should be called `Consensus` for consistency. From my understanding the refund type is used to broadly describe the tally outcome.

Reply:
I think we don’t need to distinguish between refund types. The solver always gets refunded all the funds that weren’t consumed. No actions change the amount refunded, the only change the amounts consumed.

In addition there will never be a full refund, we’ll always consume a base fee of X gas in the tally phase (prolly 1 teragas) to prevent attacks where a malicious actor posts a lot of DRs that all time out without incurring costs beyond posting the DR on the contract.

Reply:
They have been removed.

Reply:
The refund enum can be removed entirely right?

Reply:
Yes I forgot to whoops

Reply:
fixed :)